meta-xt-control-domain/conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "xt-control-domain"
 BBFILE_PATTERN_xt-control-domain := "^${LAYERDIR}/"
 BBFILE_PRIORITY_xt-control-domain = "6"
 
-LAYERSERIES_COMPAT_xt-control-domain = "dunfell"
+LAYERSERIES_COMPAT_xt-control-domain = "kirkstone"
```

Comment:
Just to clarify: it will not be compatible with `dunfell` after this change?

Reply:
After the change in that particular commit I am not 100% sure whether it still be compatible with dunfell, although an override syntax change seems to be supported by dunfell as well, (but the question is in common license renaming which is also done in current patch https://docs.yoctoproject.org/3.3.2/ref-manual/migration-3.3.html#single-version-common-license-file-naming). But, after the changes in the whole PR it definitely won't, I am afraid.

Reply:
Are you asking because common is used by other product's which cannot be kirkstone based right now? If so can these be fixed on revs prior to this series?

Reply:
"+"